python/tree_plotting.py

```diff
@@ -696,6 +737,12 @@ def draw_plot(plotDict, plotname, lumi, year, logY=True, logX=False, plotData=Fa
     latexCMS.DrawLatex(0.13,0.86+expoffset,"CMS");
     latexCMSExtra.DrawLatex(0.13,0.815+expoffset, cmsExtra);
 
+  #latexCorrelation = ROOT.TLatex()
```

Comment:
are these also commented for 2D template fit?

Reply:
Yes, these are used in a very specific plotting configuration and it wasn't worth it making a flag for that but also it wasn't worth deleting them altogether, since they may prove useful for additional tests.